runtime/bindings-generator.cs

```diff
@@ -2650,6 +2650,20 @@ static IEnumerable<FunctionData> GetFunctionData (bool isDotNet = false)
 				}
 			);
 
+			data.Add (
+				new FunctionData {
+					Comment = " // bool func (Vector4, IntPtr, IntPtr)",
+					Prefix = "simd__",
+					Variants = Variants.NonStret,
+					ReturnType = Types.Bool,
+					Parameters = new ParameterData [] {
+						new ParameterData { TypeData = Types.Vector4 },
+						new ParameterData { TypeData = Types.NativeHandleType },
+						new ParameterData { TypeData = Types.NativeHandleType },
+					},
+				}
+			);
+
```

Comment:
What does this section do? 😯

Reply:
@tj-devel709 A couple of the new selectors added for this PR have the marshal directive, which prompts the generator to generate a Pinvoke. Most of the pinvokes were to signatures that were already incoporated in bindings.generator.cs, but for the GetCameraRelativePosition selector it was to a signature that wasn’t accounted for. That’s what this is trying to do here, but its a wip (its the reason for why the build is failing now..)